svm-fuzz-harness/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+use prost::Message;
+use solana_svm_fuzz_harness_fixture::proto::InstrContext as ProtoInstrContext;
+use solana_svm_fuzz_harness_instr::execute_instr_proto;
+use std::ffi::c_int;
```

Comment:
Agave's rustfmt will yell about these being separate. Can you group them together under a single `use`?

```suggestion
use {
    prost::Message,
    solana_svm_fuzz_harness_fixture::proto::InstrContext as ProtoInstrContext,
    solana_svm_fuzz_harness_instr::execute_instr_proto,
    std::ffi::c_int,
};
```

Reply:
done